minecraft_servers/cragenetwork/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "server_name": "cragenetwork",
+  "nice_name": "Crage Network",
+  "direct_ip": "laby.cragenetwork.com",
+  "social": {
+    "web": "https://cragenetwork.com",
+    "web_shop": "https://cragenetwork.com/store",
+    "web_support": "https://cragenetwork.com/support",

```

Comment:
Cannot be behind login; has to be public (public FAQs or public contact form) or remove it

Reply:
Updated.